hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/attributes/DefaultTraceAttributeReader.java

```diff
@@ -10,7 +10,7 @@
 import org.hypertrace.core.datamodel.Event;
 import org.hypertrace.core.datamodel.StructuredTrace;
 
-class DefaultTraceAttributeReader implements TraceAttributeReader {
+class DefaultTraceAttributeReader implements TraceAttributeReader<StructuredTrace, Event> {
```

Comment:
It might be worth having `TraceAttributeReader<StructuredTrace, Event>` be its own interface for brevity. If it were TS, I'd create an alias. Up to you

Reply:
I miss aliases in Java